flutter/lib/src/event_processor/screenshot_event_processor.dart

```diff
@@ -1,8 +1,9 @@
 import 'dart:async';
 import 'dart:math';
 import 'dart:typed_data';
-import 'dart:ui';
+import 'dart:ui' as ui;
```

Comment:
Isn't it easier and less code to use a named import for the widget lib? something like that

```dart
import 'package:flutter/widgets.dart' as widget;

if (_options.attachScreenshotWhenResumed &&
    widget.WidgetsBinding.instance.lifecycleState != AppLifecycleState.resumed) {
  _options.logger(SentryLevel.debug,
      'Only attaching screenshots when application state is resumed.');
  return event;
}
```